style.css

```diff
@@ -88,3 +97,305 @@
 .social-icons-set img {
   margin: 0 9px;
 }
+
+.works-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 80px;
+  padding: 120px 0;
+}
```

Comment:
- [ ] Please as mentioned by the previous reviewer there is a difference between the works section background color and the color in the Figma design. **HIT:** add `background: #E5E5E5;` to the `.works-container` class.

![227657272-2e720737-a716-43cc-8ad8-f62358f01eaf](https://user-images.githubusercontent.com/91142432/227689507-2b611be0-7a9c-4948-971b-d4ceeed3cea6.png)